staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -242,9 +254,11 @@ func (d *Helper) evictPods(pods []corev1.Pod, policyGroupVersion string, getPodF
 				errors = append(errors, err)
 			}
 		case <-globalTimeoutCh:
-			return fmt.Errorf("drain did not complete within %v", globalTimeout)
+			errors = append(errors, fmt.Errorf("drain did not complete within %v", globalTimeout))
```

Comment:
This change is a nice-to-have, I can remove it.  Currently, all other errors are swallowed if we hit the global timeout.  This will allow us to see the errors for other pods that aren't blocked on PDBs.